src/store/middleware/model-poller.ts

```diff
@@ -132,12 +133,24 @@ export const modelPollerMiddleware: Middleware<
           return;
         }
 
+        const isProduction = import.meta.env.PROD;
+        const analyticsEnabled = window.jujuDashboardConfig?.analyticsEnabled;
```

Comment:
This should get the value from the redux store.
```suggestion
        const analyticsEnabled = getAnalyticsEnabled(reduxStore.getState());
```

Reply:
Done